src/ansys/fluent/core/streaming_services/events_streaming.py

```diff
@@ -27,8 +27,10 @@ class SolverEvent(Enum):
 
     TIMESTEP_STARTED = "TimestepStartedEvent"
     TIMESTEP_ENDED = "TimestepEndedEvent"
+    TIMESTEP_ENDED_SYNC = "TimestepEndedSyncEvent"
```

Comment:
@mkundu1 Do we need to create a documentation issue?  The user will need to know the difference between the new and existing events. Event type details are in general missing in the API reference, so could be added here (in a separate task).

Reply:
As discussed, made the original `ITERATION_ENDED` and `TIMESTEP_ENDED` as sync events, no doc change is required